bitnami/external-dns/templates/configmap.yaml

```diff
@@ -1,10 +1,16 @@
-{{- if and (eq .Values.provider "designate") .Values.designate.customCA.enabled }}
+{{- if and (eq .Values.provider "designate") .Values.designate.customCA.enabled | or (and (eq .Values.provider "rfc2136") .Values.rfc2136.rfc3645Enabled) }}
```

Comment:
It may make sense to do something in the `_helpers.tpl` file like we do with the `createSecret` helper.